deploy/operator/after_upgrade.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+export UPGRADE_ASSISTED_OPERATOR="false"
+export ASSISTED_STOP_AFTER_AGENT_DISCOVERY="true"
```

Comment:
I think that it would be more intuitive if we will inverse it. Similar to what we have in assisted-test-infra. There's `deploy_nodes` and `deploy_nodes_with_install`
```suggestion
export ASSISTED_WAIT_FOR_INSTALLATION="false"
```
WDYT? @flaper87